### DISCUSSION TITLE
Help: New Setup to Connect To HyperHDR Docker Over Wifi

### DISCUSSION THREAD

**Discussion by definitelynotpaul** [awawa-dev/HyperHDR#955] [Q&A] [answered]
My previous HyperHDR setup was be controlled by a wireless LED controller running WLED and HyperHDR running in a docker container on my unraid server. The majority of the content I watch is via Google TV built into the TV so I've been using the android grabber to send the capture to HyperHDR over the network. This has worked flawlessly and I'd like to replicate it. However, my new TV is much larger and I need to use a new controller as my old one doesn't support the amount of LEDs needed for the new TV. 

If I purchase the Pimoroni Plasma Stick 2040 W, can I use it in the same way via HyperSerialPico? If not, what is the best setup to replicate what I had before?

**Comment by awawa-dev** [comment]
No, HyperSerialPico nor HyperSPI dont work over wifi. How many LEDs do you have and what model of ESP board with WLED can't handle it? ESP32 should handle it, with Esp8266 and WLED there are huge problems since around version 0.14-0.14.1 and Esp8266 itself is already an very outdated design.

**Reply by definitelynotpaul** [reply]
My previous setup was roughly 220 LEDs. The new TV is much bigger and should be around 270-280 LEDs. The previous controller can only support "250".

I've realized WLED is probably not the way to go at this point, and I'm happy to switch to a more updated solution, but I still need to use the android grabber as most of the content is running on the native android TV so there's no other way to grab it. 

Can I still use the android grabber if I switch to running HyperHDR on a rPi 4 connected to an rp2040 to drive the LEDs?

**Reply by awawa-dev** [reply, marked as answer]
As far I know from the users reports, HyperHDRs protobuffer client is still working with the android grabber: just need to be enabled. I recommend Adafruit boads:Adafruit RP2040 ItsyBitsy  (one output so no multi-segments) or more future proof (8 outputs) Scorpio. Can be used with RPi using wired connection flashed with HyperSerialPico (USB connection) or HyperSPI (short cables connecting GPIOs). If you want wifi you must use generic esp32 and wled.

**Reply by definitelynotpaul** [reply]
I will go this route. I had already ordered and received a Scorpio before realizing it's not a Pico W board.

Thank you for your help!
